e2e-tests/tests/test_services_integration.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from common.methods import analyzer_supported_entities
+
+
+# TODO: [ADO-2763] Stub, remove and replace with real integration scenarios.
```

Comment:
Is this comment still relevant?

Reply:
If it will be merged before your change then yes, and you will have to replace it with real scenarios